toxcore/TCP_client.h

```diff
@@ -15,6 +15,7 @@
 #include "logger.h"
 #include "mem.h"
 #include "mono_time.h"
+#include "net_profile.h"
```

Comment:
here too.

Reply:
Where else in toxcore's headers do we replace headers with forward declarations?